src/macros.rs

```diff
@@ -327,3 +327,95 @@ macro_rules! template {
     };
     () => {}
 }
+
+#[macro_export]
+macro_rules! labels {
+
+    ($($tail:tt)+) => (
+        labels_sep_by!(" "; $($tail)*)
+    );
+
+}
+
+#[macro_export]
+macro_rules! labels_sep_by {
+
+    (@inner_expand $sep:expr; $tmpl:ident $item:expr) => {
+        $sep.render_once($tmpl);
+        $item.render_once($tmpl);
+    };
+
+    (@inner_expand $sep:expr; $tmpl:ident $item:expr => $should_include:expr) => {
+        if $should_include {
+            $sep.render_once($tmpl);
+            $item.render_once($tmpl);
+        }
+    };
+
+    (@inner_expand $sep:expr; $tmpl:ident $item:expr, $($tail:tt)+) => {
+        $sep.render_once($tmpl);
+        $item.render_once($tmpl);
+        labels_sep_by!(@inner_expand $sep; $tmpl $($tail)*);
+    };
+
+    (@inner_expand $sep:expr; $tmpl:ident $item:expr => $should_include:expr, $($tail:tt)+) => {
+        if $should_include {
+            $sep.render_once($tmpl);
+            $item.render_once($tmpl);
+        }
+        labels_sep_by!(@inner_expand $sep; $tmpl $($tail)*);
+    };
+
+    // entries
+
+    ($sep:expr; $item:expr) => {{
+
+        use $crate::Template;
+        use $crate::RenderOnce;
```

Comment:
Instead of importing and calling, just use the fully qualified syntax: `$crate::RenderOnce::render(&$item, tmpl)`

Otherwise, you run the risk of running into conflicting functions.